### PR TITLE
Fct 210 sample data models attribute plain enum type

### PR DESCRIPTION
**Pull request by tylermorrisford**
Adds the `attribute-plain-enum-value` model and spec.

### CODE REVIEW

File: .changeset/red-ducks-battle.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-test-data/plain-enum-value': minor
```
Comment (jaikamat) [inline .changeset/red-ducks-battle.md:2]: Does this changeset have to be regenerated now that the directory name has changed? Honest question, I'm not sure :)

Comment (tdeekens) [reply .changeset/red-ducks-battle.md:2]: Package name matters here. What has changed?

Comment (tdeekens) [reply .changeset/red-ducks-battle.md:2]: It has to change as pkg name changed.

Comment (tylermorrisford) [reply .changeset/red-ducks-battle.md:2]: Yes, slight whiff there on my part.

File: .changeset/olive-jars-tap.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-test-data/attribute-plain-enum-value': minor
+---
+
+Renamed `plain-enum-value` to `attribute-plain-enum-value`
```
Comment (stephsprinkle) [inline .changeset/olive-jars-tap.md:5]: I don't think we want two changesets here, better to update the previous one.

File: .changeset/red-ducks-battle.md (hunk 1 above)
Comment (stephsprinkle) [inline .changeset/red-ducks-battle.md:2]:
```suggestion
'@commercetools-test-data/attribute-plain-enum-value': minor
```

File: .changeset/red-ducks-battle.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+---
+
+create plain-enum-value model
```
Comment (stephsprinkle) [inline .changeset/red-ducks-battle.md:5]:
```suggestion
Add model for `AttributePlainEnumValue`.
```

**Review by stephsprinkle — APPROVED** [review]
Everything looks good to me, just some changeset suggestions

**Pull request merged**